pgx-version-updater/src/main.rs

```diff
@@ -251,30 +264,32 @@ fn main() {
             let docstring = doc.to_string();
 
             std::thread::spawn(move || {
-                stdin
-                    .write_all(docstring.as_bytes())
-                    .expect("Failed to write to stdin");
+                stdin.write_all(docstring.as_bytes()).expect("Failed to write to stdin");
             });
 
             let child_output = child.wait_with_output().expect("Failed to read stdout");
-            let mut diff_output = String::new();
-
-            // Using an iterator here, so we need to keep track of certain things for proper output
-            let mut line_ctr = 0;
-            for line in child_output.stdout.lines().skip(2) {
-                let line = line.unwrap();
 
-                match line.chars().nth(0).unwrap() {
-                    '-' => diff_output.push_str(format!("\n            {}", line.red()).as_str()),
-                    '+' => diff_output.push_str(format!("\n            {}", line.green()).as_str()),
-                    _ => diff_output.push_str(format!("\n           {line}").as_str()),
+            // Loop through all lines of the diff command output, if any. First 2 lines
+            // from the diff output above will produce irrelevant information, so we
+            // will skip it.
+            let mut diff_output = String::new();
+            for output_line in child_output.stdout.lines().skip(2) {
+                if let Ok(line) = output_line {
+                    match line.chars().nth(0) {
+                        Some(c) if c == '-' => {
```

Comment:
We can match directly on a value instead of binding it to a pattern, especially if we don't actually care about the value afterwards.
```suggestion
                        Some('-') => {
```